From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Fred Silberberg <frsilb@microsoft.com>
Date: Thu, 21 Jul 2022 10:02:32 -0700
Subject: [PATCH] Exclude the razor syntax generator from source build (#305)

This is a manually-run tool to update syntax nodes and doesn't need to be built by source-build. Fixes https://github.com/dotnet/razor-compiler/issues/304.

Backport: https://github.com/dotnet/razor-compiler/pull/305
---
 src/tools/RazorSyntaxGenerator/RazorSyntaxGenerator.csproj | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/tools/RazorSyntaxGenerator/RazorSyntaxGenerator.csproj b/src/tools/RazorSyntaxGenerator/RazorSyntaxGenerator.csproj
index 924379ac..f213af41 100644
--- a/src/tools/RazorSyntaxGenerator/RazorSyntaxGenerator.csproj
+++ b/src/tools/RazorSyntaxGenerator/RazorSyntaxGenerator.csproj
@@ -9,6 +9,7 @@
     <IsPackable>false</IsPackable>
     <!-- No need to track public APIs of this tool. -->
     <AddPublicApiAnalyzers>false</AddPublicApiAnalyzers>
+    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
   </PropertyGroup>
 
 </Project>
